docs/content/development/datasketches-aggregators.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: doc_page
+---
+
+## DataSketches aggregator
+Druid aggregators based on [datasketches]()http://datasketches.github.io/) library. You would ingest one or more metric columns using either `sketchBuild` or `sketchMerge` aggregators. Then, at query time, you can use `sketchMerge` with appropriate post aggregators described below. Note that sketch algorithms are approxiate, see details in the [datasketches doc](http://datasketches.github.io/docs/theChallenge.html).
```

Comment:
I *think* a theta sketch, which is a more general version of KMV is being done, is that true?

if so, can we call the aggregators `thetaIngest` and `theta`?